### PR TITLE
feat(breadcrumbs): add breadcrumbs

### PR DESCRIPTION
**Pull request by oysteinsigholt**
screenshot from 2017-06-30 14-44-22

### CODE REVIEW

**Comment by tipsy** [comment]
Are you sure this is how it looks in the design?

**Comment by oysteinsigholt** [comment]
I havent received any design for breadcrumbs, so this is just something basic to expand upon. Note that the clickable links in the photo are unstyled as there is no link style yet.

File: components/collections/tn-breadcrumbs.scss [hunk 1]
```diff
@@ -0,0 +1,23 @@
+@import "tn-breadcrumbs-mixins";
+@import "../_variables/common";
+
+/**
+ * Breadcrumbs to indicate the user's current position in a hierarchical structure
+ *
+ * @atom Breadcrumbs
```
Comment (pkkummermo) [inline components/collections/tn-breadcrumbs.scss]: Make it a molecule.

File: components/collections/tn-breadcrumbs.scss [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+ * @section Collections
+ * @markup
+ *  <ol class="tn-breadcrumbs">
+ *   <li class="tn-breadcrumb"><a href="#" class="tn-a">Home</a></li>
```
Comment (pkkummermo) [inline components/collections/tn-breadcrumbs.scss]: Pull in `.tn-a` from master and add to a-tag.

Comment (oysteinsigholt) [inline components/collections/tn-breadcrumbs.scss]: Not sure what you mean by this, can you elaborate?

Comment (pkkummermo) [inline components/collections/tn-breadcrumbs.scss]: Link-styling was merged to master :) If you rebase on top of that you can add the `tn-a`-class to the `a`-tags inside the markup example

Comment (oysteinsigholt) [inline components/collections/tn-breadcrumbs.scss]: They should already have the tn-a class, so now that i have rebased everything looks better :)
screenshot from 2017-06-30 16-24-32

**Comment by oysteinsigholt** [comment]
Should the breadcrumbs be larger and have a --small version?

**Comment by tipsy** [comment]
> Should the breadcrumbs be larger and have a --small version?

No, I don't think that's needed

**Pull request merged**